integration/package.json

```diff
@@ -1,3 +1,23 @@
 {
-  "type": "module"
+  "name": "integration-tests",
+  "version": "0.0.0-local-version",
+  "description": "deps needed for integration tests",
+  "private": true,
+  "type": "module",
+  "scripts": {},
+  "dependencies": {
+    "@cloudflare/kv-asset-handler": "^0.3.0",
+    "@cloudflare/workers-types": "^4.20230518.0",
+    "@vanilla-extract/css": "^1.10.0",
+    "cross-env": "^7.0.3",
+    "express": "^4.17.1",
+    "isbot": "^3.6.8",
+    "npm-run-all": "^4.1.5",
+    "postcss": "^8.4.19",
+    "postcss-import": "^15.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "tailwindcss": "^3.3.0",
+    "typescript": "^5.1.0"
```

Comment:
If something is in the root, do we need to dup it here?  I think its just `react`/`react-dom`/`typescript`

Reply:
I wanted to make them explicit here as I think we should at some point refactor to make them devDeps in the appropriate packages and root `package.json` only needs things like ESLint, rollup, Prettier, ...